index.js

```diff
@@ -1,6 +1,61 @@
 /* jshint node: true */
 'use strict';
 
+var path = require('path');
+var Funnel = require('broccoli-funnel');
+var mergeTrees = require('broccoli-merge-trees');
+var VersionChecker = require('ember-cli-version-checker');
+
 module.exports = {
-  name: 'ember-a11y'
+  name: 'ember-a11y',
+
+  init: function() {
+    if (this._super.init) {
+      this._super.init.apply(this, arguments);
+    }
+
+    var versionChecker = new VersionChecker(this);
+    versionChecker.for('ember-cli', 'npm').assertAbove('0.2.0');
+
+    var emberVersion = versionChecker.for('ember', 'bower');
```

Comment:
We're shipping `ember-source` with 2.11 per current plans so let's go ahead and account for it.

`var emberVersion = versionChecker.for('ember-source', 'npm');` => `glimmer` every time if it's included that way.